community/lexicon/forms/question.json

```diff
@@ -0,0 +1,89 @@
+{
+  "lexicon": 1,
+  "id": "community.lexicon.forms.question",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "Record declaring a question in a form.",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": [
+          "title",
+          "questiontype"
+        ],
+        "properties": {
+          "title": {
+            "type": "string",
+            "maxLength": 100,
+            "description": "The title of the question."
+          },
+          "description": {
+            "type": "string",
+            "maxLength": 2000,
+            "description": "Description of the question."
+          },
+          "questiontype": {
+            "type": "ref",
+            "ref": "community.lexicon.forms.question#questiontype",
+            "description": "The type of question."
+          },
+          "questionoptions": {
+            "type": "array",
+            "description": "The options for a question with multiple answers.",
+            "items": {
+              "type": "ref",
+              "ref": "community.lexicon.forms.question#questionoption"
+            }
+          },
+          "required": {
+            "type": "boolean",
+            "description": "Indicates if this question must be answered.",
+            "default": false
+          }
+        }
+      }
+    },
+    "questiontype": {
```

Comment:
We should probably use `camelCase` for fields. This is the casing pattern Bluesky is using in atproto / bsky.app